server/bin/www

```diff
@@ -31,6 +31,21 @@ server.listen(port);
 server.on("error", onError);
 server.on("listening", onListening);
 
+/**SocketIO
+ *
+ */
+var socket = require("socket.io");
+var io = socket(server);
+
+io.on("connection", function(socket) {
+  console.log("a user connected", socket.id);
+  socket.on("disconnect", function() {
+    console.log("user disconnected");
+  });
+});
+
+io.attach(server);
```

Comment:
You can remove this line now, because you're already passing the server on the constructor at line 38. You only do one or the other.

Reply:
I see, thanks Aecio